### ISSUE TITLE
Frontend member say multiline comments are not allowed... but this repo says they are fine!

### ISSUE THREAD

**Issue by trisrael** [AltSchool/javascript#13]
Trying to enforce no multiline comments is difficult when this repo says they are absolutely fine and how to use them correctly.

I don't know the argument for removing multiline comments other than perhaps consistency ( and possibly readability ) so I don't feel comfortable opening a PR to remove it ( or reword it )

[our Javascript multiline comments section](https://github.com/AltSchool/javascript#17.1)

**Comment by matthewconstantine** [comment]
I don't agree with AirBnb on this one. We should allow blocks of `//` comments. Most editors make it easier to write these than multiline.

**Comment by DeviateFish** [comment]
I think we should allow blocks of `//`, but I don't think they should be the only form of comment allowed.

YUIdocs, JSdoc, etc all rely on `/** */` style comments for documentation, specifically.  I think, for documentation purposes (especially around apis/components/etc), we should pick a standard (it was YUIDocs, I thought?) and not choose style rules that conflict with it.

**Comment by matthewconstantine** [comment]
@DeviateFish Agreed 100%.

Also, `//` comments place less burden on the coders. We don't want to slow people down just because they extend a comment to two lines, and now have to change the comment style. Ugh.

**Comment by alexgb** [comment]
Let's update docs so that `//` style multiline comments are ok as well as `/* */` block comments for yuidoc parsing. I'd prefer to default to `//` in all cases and only use block comments when needed for doc generation.
